### PR TITLE
Support style in month title

### PR DESCRIPTION
**Pull request by zhenleiji**
## What I did
- Added the possibility to customise the month title from `tsquare_titleTextStyle` attribute (default value, same as before: `@style/CalendarTitle`)
- Removed unnecessary `tsquare_titleTextColor` as we can customise from the style
- Example how to use: `sample/src/main/res/layout/dialog_customized.xml`

### CODE REVIEW

File: library/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!-- Copyright 2012 Square, Inc. -->
 <resources>
   <dimen name="calendar_day_headers_paddingbottom">4dp</dimen>
-  <dimen name="calendar_month_topmargin">16dp</dimen>
-  <dimen name="calendar_month_title_bottommargin">4dp</dimen>
+  <dimen name="calendar_month_title_paddingtop">16dp</dimen>
```
Comment (edenman) [inline library/src/main/res/values/dimens.xml]: I don't think we should rename these.  It's pretty likely that people are overriding these values just by defining their dimens with the same name, and if we change the name, their customizations will silently fail.

File: library/src/main/res/layout/month.xml [hunk 2]
```diff
@@ -6,15 +6,6 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     >
-  <TextView
```
Comment (edenman) [inline library/src/main/res/layout/month.xml]: let's add a comment here explaining that the title view will be added programmatically

File: library/src/main/res/values/dimens.xml (hunk 1 above)
Comment (zhenleiji) [reply library/src/main/res/values/dimens.xml]: I didn't think about it. Nice catch! I am changing now.

File: library/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -3,6 +3,8 @@
   <dimen name="calendar_day_headers_paddingbottom">4dp</dimen>
   <dimen name="calendar_month_topmargin">16dp</dimen>
   <dimen name="calendar_month_title_bottommargin">4dp</dimen>
+  <dimen name="calendar_month_title_paddingtop">16dp</dimen>
```
Comment (edenman) [inline library/src/main/res/values/dimens.xml]: I think you also need to remove these and change their uses over to the old dimens

**Comment by edenman** [comment]
LG.  Just need you to sign the [Square CLA](https://docs.google.com/forms/d/e/1FAIpQLSeRVQ35-gq2vdSxD1kdh7CJwRdjmUA0EZ9gRXaWYoUeKPZEQQ/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1).  Please comment here when you've done so (I don't get notified of new signatures)

**Comment by zhenleiji** [comment]
Done! Thanks for the quick review!

**Comment by edenman** [comment]
Thanks for the contribution! :)

**Comment by edenman** [comment]
released as 1.7.5

**Pull request merged**